miri

```diff
@@ -281,8 +281,10 @@ find_sysroot() {
 case "$COMMAND" in
 install)
     # "--locked" to respect the Cargo.lock file if it exists.
-    $CARGO install $CARGO_EXTRA_FLAGS --path "$MIRIDIR" --force --locked "$@"
-    $CARGO install $CARGO_EXTRA_FLAGS --path "$MIRIDIR"/cargo-miri --force --locked "$@"
+    # Install binaries to the miri toolchain's sysroot so they do not interact with other toolchains
+    echo Binaries should be installed to $SYSROOT
```

Comment:
"should be"? This sounds like you are telling the user what they should do, but that's probably not the intent here? "will be installed" sounds more clear?

Reply:
Oops that's from debugging and I didn't delete it